rhscl-postgresql-apb/apb.yml

```diff
@@ -0,0 +1,31 @@
+id: 8a049637-8c70-4434-993c-a15c3d310513
+name: rhscl-postgresql-apb
+image: ansibleplaybookbundle/rhscl-postgresql-apb
+description: SCL PostgreSQL apb implementation
+bindable: true
+async: optional
+metadata:
+  displayName: "Red Hat SCL Postgresql"
+  longDescription: "An apb that deploys postgresql 9.4 or 9.5."
+  imageUrl: "https://wiki.postgresql.org/images/a/a4/PostgreSQL_logo.3colors.svg"
+  documentationUrl: "https://www.postgresql.org/docs/"
+parameters:
+  - name: namespace
```

Comment:
I agree we have to deal with collisions somehow. I think maybe there has to be more logic here on the broker/catalog side possibly? Maybe within the apb's as well.... I can see wanting to deploy to a new apb app to an existing namespace. So it's not just enough to check if a namespace doesn't exist; something will need to look and say this app is already deployed to this namespace and if you try again it's probably going to fail...

Then again we added a timestamp to rds deployments in the summit demo and they didn't collide. So maybe there's something we can try to do with unique suffixes or something without getting too ugly. We all probably need to get together and discuss what and how we want to support such scenarios.